### DISCUSSION TITLE
Desirability

### DISCUSSION THREAD

**Discussion by Ranmyaku2** [Keriew/augustus#779] [General]
I was wondering  if you had considered offering a chart or something of desirability values. With new features and buildings you have added to caesar 3 in Augustus  it would be very helpful to know what might help bring up and area without spamming a bunch of stuff and then looking at the desirability overlay etc. Also being able to see if original game desirability  features have changed.  Perhaps this could  be added to the manual?

**Comment by Ouaz** [comment]
> I was wondering if you had considered offering a chart or something of desirability values.

There's a great fansite with some desirability values (with range and effect): http://mmxl.wz.cz/c3a/structures.html

I think desirability changes are mentioned somewhere in the changelog's hundreds of lines. XD

**Reply by Ranmyaku2** [reply]
oh thanks so much wasn't aware of this site. I appreciate the help.
